test/content-negotiation-test.js

```diff
@@ -0,0 +1,61 @@
+var test = require('tape');
+var Hapi = require('hapi');
+var server = new Hapi.Server();
+var routes = require('../routes.js');
```

Comment:
Since we're using node 6 it might be a better idea to express that ee don't intend to re-assign to these variables by declaring them as `const`.

Also, ".js" isn't required in `require` calls.